atomic_reactor/plugins/pre_fetch_maven_artifacts.py

```diff
@@ -147,8 +153,10 @@ def process_by_nvr(self, nvr_requests):
                 # not contain a trailing slash, which causes the last dir to
                 # be dropped.
                 url = maven_build_path + '/' + maven_file_path
+                checksum_type = koji.CHECKSUM_TYPES[build_archive['checksum_type']]
```

Comment:
Why are we using `koji.CHECKSUM_TYPES` here - and then just `CHECKSUM_TYPES` below?